data/campaigns/Winds_of_Fate/units/Jaguar.cfg

```diff
@@ -0,0 +1,63 @@
+#textdomain wesnoth-wof
+
+[unit_type]
+    id=Jaguar
+    name= _ "Jaguar"
+    race=monster
+    image="units/jaguar.png"
+    hitpoints=45
+    movement_type=woodland
+    [defense]
+        forest=40
+        village=50
+    [/defense]
+    movement=8
+    experience=100
+    level=2
+    alignment=neutral
+    advances_to=null
+    {AMLA_DEFAULT}
+    undead_variation=wolf
+    cost=23
+    usage=fighter
+    description= _ "The jaguar is a predator that lives in tropical areas. Some of the larger and more aggressive jaguar routinely kill boars, and a few have been known to attempt human-sized prey."
+    {NOTE_AMBUSH}
```

Comment:
This is recent (added in PR #5811), but `{NOTE_AMBUSH}` is now deprecated and the note is added automatically.